### PR TITLE
Design review: alchemist

### PR DESCRIPTION
**Pull request by plof27**
I made some changes, but also left comments with questions/concerns

### CODE REVIEW

File: rulebook/src/character-options/classes/alchemist/features.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 **Effects:** As an alchemist, you gain acccess to concoctions. These are potent 2 AP, 3 essence powers that have a 3 tile range, and affect a 1 tile radius around your target. When you use a concoction, you provoke an attack of opportunity. Even if you miss with your attack, they apply half their effect.
 
-When you gain this feature, choose a subclass. When selecting concoctions, you must know more concoctions from that subclass than from any other subclass.
+When you gain this feature, choose a Pursuit. When selecting concoctions, you must know more concoctions from that subclass than from any other subclass.
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/classes/alchemist/features.md:32]:
```suggestion
When you gain this feature, choose a Pursuit, which defines your subclass. When selecting concoctions, you must know more concoctions from that subclass than from any other subclass.
```

File: rulebook/src/character-options/classes/alchemist/features.md [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 **Effects:** As an alchemist, you gain acccess to concoctions. These are potent 2 AP, 3 essence powers that have a 3 tile range, and affect a 1 tile radius around your target. When you use a concoction, you provoke an attack of opportunity. Even if you miss with your attack, they apply half their effect.
 
-When you gain this feature, choose a subclass. When selecting concoctions, you must know more concoctions from that subclass than from any other subclass.
+When you gain this feature, choose a Pursuit, which defines your subclass. When selecting concoctions, you must know more concoctions from that subclass than from any other subclass.
```
[the remark this answers is not in the document]
Comment (plof27) [reply rulebook/src/character-options/classes/alchemist/features.md:32]: Resolved: adding back original subclasses

File: rulebook/src/character-options/classes/alchemist/features.md [hunk 3]
```diff
@@ -29,7 +29,7 @@
 
 **Effects:** As an alchemist, you gain acccess to concoctions. These are potent 2 AP, 3 essence powers that have a 3 tile range, and affect a 1 tile radius around your target. When you use a concoction, you provoke an attack of opportunity. Even if you miss with your attack, they apply half their effect.
 
-When you gain this feature, choose a subclass. When selecting concoctions, you must know more concoctions from that subclass than from any other subclass.
+When you gain this feature, choose a Pursuit. When selecting concoctions, you must know more concoctions from that subclass than from any other subclass.
 
 ### Preparations
```
[the remark this answers is not in the document]
Comment (plof27) [reply rulebook/src/character-options/classes/alchemist/features.md:34]: Resolved: Although it wasn't the original intent, this proposed level 1 structure *is* better. Going to go with this moving forward

[the remark this answers is not in the document]
Comment (plof27) [reply rulebook/src/character-options/classes/alchemist/features.md:34]: Done in 1da35cc197be33d95fb2c5d251d5951c3563bb7b

File: rulebook/src/character-options/classes/alchemist/features.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (plof27) [reply rulebook/src/character-options/classes/alchemist/features.md:32]: After talking, we decided to cut subclasses entirely for complexity. After adding back drawbacks, there was just too much to pick at level 1.

File: rulebook/src/character-options/feats/exploration-feats.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # Exploration Feats
 
+### Epiphany
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/feats/exploration-feats.md:3]:
```suggestion
### Inspired
```
These feats all follow "adjective that describes a person" naming scheme.

File: rulebook/src/character-options/classes/conduit/combat_feats.md [hunk 5]
```diff
@@ -93,3 +93,7 @@ For clarification, the source of the attacks for chained charges is the origin o
 **Prerequisites:** Buffer Overflow class feature
 
 **Effects:** Lose access to your Buffer Overflow class feature: your charges can no longer be chained. You may now use charges as attacks of opportunity, and may choose to use this action to channel a charge into your weapon before using the Strike action as part of an attack of opportunity.
+
+## Discharge
+
+**Effects:** Whenever you use a charge, you may **discharge**. If you do, remove all stacks of absorption from yourself. The charge deals additional electric damage equal to the stacks of absorption removed.
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/classes/conduit/combat_feats.md:99]:
```suggestion
**Effects:** Whenever you use a charge, you may remove all stacks of absorption from yourself. The charge deals additional electric damage equal to the stacks of absorption removed.
```
Keywording was pointless.

File: rulebook/src/character-options/classes/alchemist/features.md [hunk 6]
```diff
@@ -115,100 +121,108 @@ This metamagic choice is more complex, and requires careful thought about how an
...
-At the start of combat, roll to select a random ailment. Your concoction applies stacks of that ailment instead.
+#### Arcing 
 
-Once this metamagic has been used, reroll to select the next ailment.
+Your concoction now jumps from creature to creature.
+At each jump, select the closest creature (within 2 tiles) of the previous one.
+This chain can contain up to three targets total.
+Creatures can only be part of each chain once; you cannot return to them.
+In the event of a tie, the caster may choose which creatures to affect.
+Halve the efficacy of spells altered in this way.
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/classes/alchemist/features.md:181]:
```suggestion
Halve the efficacy of concoctions altered in this way.
```

File: rulebook/src/character-options/classes/alchemist/features.md [hunk 7]
```diff
@@ -115,100 +121,108 @@ This metamagic choice is more complex, and requires careful thought about how an
...
+#### Hose
 
-#### Eclectic
+Your concoction now affects a 6 tile line in front of you. Halve the efficacy of concoctions altered in this way.
 
-At the start of combat, roll to select a random ailment. Your concoction applies stacks of that ailment instead.
+#### Arcing 
 
-Once this metamagic has been used, reroll to select the next ailment.
+Your concoction now jumps from creature to creature.
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/classes/alchemist/features.md:176]:
```suggestion
This concoction now jumps from creature to creature.
```

File: rulebook/src/character-options/classes/alchemist/features.md [hunk 8]
```diff
@@ -115,100 +121,108 @@ This metamagic choice is more complex, and requires careful thought about how an
...
+#### Spray
 
-#### Homeopathic
+Your concoction now affects a 2 tile radius cone in front of you. Halve the efficacy of concoctions altered in this way.
 
-Whenever you would apply stacks of an ailment or affliction with this concoction, instead remove that many stacks of that ailment or affliction.
+#### Hose
 
-#### Eclectic
+Your concoction now affects a 6 tile line in front of you. Halve the efficacy of concoctions altered in this way.
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/classes/alchemist/features.md:172]:
```suggestion
This concoction now affects a 6 tile line in front of you. Halve the efficacy of concoctions altered in this way.
```

File: rulebook/src/character-options/classes/alchemist/features.md [hunk 9]
```diff
@@ -115,100 +121,108 @@ This metamagic choice is more complex, and requires careful thought about how an
...
+#### Blast
 
-#### Noxious
+This concoction affects a 2 tile radius. Halve the efficacy of concoctions altered in this way.
 
-Whenever you would deal damage with this concoction, instead apply stacks of brittle equal to the damage that you would have dealt.
+#### Spray
 
-#### Homeopathic
+Your concoction now affects a 2 tile radius cone in front of you. Halve the efficacy of concoctions altered in this way.
```
Comment (alice-i-cecile) [inline rulebook/src/character-options/classes/alchemist/features.md:168]:
```suggestion
This concoction now affects a 2 tile radius cone in front of you. Halve the efficacy of concoctions altered in this way.
```

**Pull request merged**